src/ConfigurationClientManager.ts

```diff
@@ -0,0 +1,317 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AppConfigurationClient, AppConfigurationClientOptions } from "@azure/app-configuration";
+import { ConfigurationClientWrapper } from "./ConfigurationClientWrapper";
+import { TokenCredential } from "@azure/identity";
+import { AzureAppConfigurationOptions, MaxRetries, MaxRetryDelayInMs } from "./AzureAppConfigurationOptions";
+import { isFailoverableEnv } from "./requestTracing/utils";
+import * as RequestTracing from "./requestTracing/constants";
+
+const TCP_ORIGIN = "_origin._tcp";
+const ALT = "_alt";
+const TCP = "_tcp";
+const Endpoint = "Endpoint";
```

Comment:
Could you update to the existing naming convention: https://github.com/Azure/AppConfiguration-JavaScriptProvider/blob/preview/src/featureManagement/constants.ts to make it consistent.